libminifi/src/c2/protocols/RESTProtocol.cpp

```diff
@@ -129,7 +129,7 @@ const C2Payload RESTProtocol::parseJsonResponse(const C2Payload &payload, const
   } catch (...) {
   }
 #endif
-  return std::move(C2Payload(payload.getOperation(), state::UpdateState::READ_COMPLETE, true));
```

Comment:
These generated copy-elision warnings in clang, so removed.

Reply:
Good catch. I rarely build with clang these days...